pypeit/spectrographs/gemini_gmos.py

```diff
@@ -143,7 +143,7 @@ def default_pypeit_par(cls):
 
         # Alter the method used to combine pixel flats
         par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
-        par['calibrations']['pixelflatframe']['process']['sig_lohi'] = [10.,10.]
+        # par['calibrations']['pixelflatframe']['process']['sig_lohi'] = [10.,10.]
```

Comment:
remove

Reply:
@profxj I noticed, only now, that for DEIMOS you changed this line with 
`par['calibrations']['pixelflatframe']['process']['comb_sigrej'] = 10.`

Should I do that with the other instruments as well, instead of removing?

Reply:
Ah, good catch.  Hard to know, actually, as this was never even turned on! :)

I say remove and we'll see how the defaults do on the DevSuite.

Reply:
Ok. Done!